docs/security/responsibilities.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Understanding your responsibilities when using Qiskit Runtime
+description: Learn about the management responsibilities that you have when you use Qiskit Runtime.
+---
+
+# Understanding your responsibilities when using Qiskit Runtime
+
+Review the following sections for the specific responsibilities for you and for IBM&reg; when you use Qiskit Runtime. For the overall terms of use, see [Terms and Notices](https://cloud.ibm.com/docs/overview/terms-of-use?topic=overview-terms) for IBM Cloud&reg;.
+
+## Incident and operations management
+
+Incident and operations management includes tasks such as monitoring, event management, high availability, problem determination, recovery, and full-state backup and recovery.
+
+| Task | IBM responsibilities | Your responsibilities |
+| --- | --- | --- |
+|Infrastructure health management|<ul><li>Continuously deliver new features, updates, and bug fixes as needed in a transparent manner.</li><li>Provide notifications for planned maintenance, security bulletins, or unplanned outages.</li></ul> | <ul><li>[Set preferences to receive emails](https://cloud.ibm.com/user/notifications) about service notifications.</li><li>Monitor the [IBM Cloud status](https://cloud.ibm.com/status) page for general announcements.</li></ul> |
+|Infrastructure health monitoring and notification| <ul><li>Remediate all infrastructure environment issues.</li><li> Notify customers of applicable incidents.</li></ul> | <ul><li> Ascertain the impact of each reported incident.</li><li>Engage [IBM Support](/docs/support) as required.</li></ul>|
+|Maintain IBM Cloud high availability SLA| Operate Qiskit Runtime in accordance with the [IBM Cloud Public Service Level Agreements](https://cloud.ibm.com/docs/overview?topic=overview-slas) (SLAs). | Follow high availability best practices. |
+
+## Change management
+
+Change management includes tasks such as deployment, configuration, upgrades, patching, configuration changes, and deletion.
+
+| Task | IBM responsibilities | Your responsibilities |
+| --- | --- | --- |
+| Major version upgrades | Provide availability and tools for major version updates. | Migrate to a supported version before end-of-support for an old version. |
+| API version changes | Versioning the REST APIs. | Integrate with the wanted version of REST APIs. |
+| Updates, fixes, and new features | <ul><li>Provide regular updates, bug fixes, and new features, transparently following a continuous delivery model.</li><li>Post notifications for changes that impact customers.</li></ul> | <ul><li>Set preferences to [receive email notifications](https://cloud.ibm.com/user/notifications) and monitor the [IBM Cloud status](https://cloud.ibm.com/status) page for general announcements.</li><li>Read system-generated notifications and take appropriate action.</li></ul> |
```

Comment:
Should we link here: https://quantum.cloud.ibm.com/docs/guides/latest-updates ?

Reply:
I'll link to announcements instead since they appear directly on the dashboard.